sktree/tests/test_honest_forest.py

```diff
@@ -84,10 +88,15 @@ def test_iris_multi(criterion, max_features, estimator):
     X = iris.data
     y = np.stack((iris.target, second_y)).T
     clf.fit(X, y)
-    score = mean_squared_error(clf.predict(X), y)
-    assert score < 0.5 and score < 1.0, "Failed with {0}, criterion = {1} and score = {2}".format(
-        "HForest", criterion, score
-    )
+    score = r2_score(clf.predict(X), y)
+    if honest_prior == "ignore":
+        assert score < 1.0, "Failed with {0}, criterion = {1} and score = {2}".format(
+            "HForest", criterion, score
+        )
+    else:
+        assert (
+            score > 0.0 and score < 1.0
+        ), "Failed with {0}, criterion = {1} and score = {2}".format("HForest", criterion, score)
```

Comment:
Does this actually test anything? Isn't r2_score between 0 and 1 always?

Reply:
Sure, we can have a tighter score. The current second dimension of y is not very informative, and that's probably why the score is low. I will try to optimize that.